SwiftWisdom.xcodeproj/project.pbxproj

```diff
@@ -938,21 +936,6 @@
 			shellScript = "diff \"${PODS_PODFILE_DIR_PATH}/Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [ $? != 0 ] ; then\n    # print error to STDERR\n    echo \"error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\" >&2\n    exit 1\nfi\n# This output is used by Xcode 'outputs' to avoid re-running this script phase.\necho \"SUCCESS\" > \"${SCRIPT_OUTPUT_FILE_0}\"\n";
 			showEnvVarsInLog = 0;
 		};
-		77882022D65A5916A7FFD999 /* [CP] Copy Pods Resources */ = {
```

Comment:
Why did this change? Did you re-run `pod install` If so, you could probably check in the `Podfile.lock`

Reply:
I did run `pod install` to generate the workspace

Reply:
Hmmm, since that made THIS change, didn't it also update `Podfile.lock`? You should rerun it and check in the lock file if it changed.

Reply:
Looks like the lock file did not change.